src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -59,31 +60,10 @@
 from sentry.snuba.dataset import Dataset, EntityKey
 from sentry.snuba.metrics import TS_COL_GROUP, TS_COL_QUERY, get_intervals
 from sentry.snuba.sessions_v2 import SNUBA_LIMIT, QueryDefinition, finite_or_none, get_timestamps
-from sentry.utils.snuba import raw_snql_query
+from sentry.utils.snuba import bulk_snql_query, raw_snql_query
 
 logger = logging.getLogger(__name__)
 
-#: Referrers for snuba queries
-#: Referrers must be searchable, so no string interpolation here
```

Comment:
For what it's worth, I wrote this sentence. the usecase for referrers is that ops can search for them and immediately understand what is causing those queries. I feel like with sessions-v2 that's not possible as you can do all kinds of queries with that API. Ideally we would accept referrers from the frontend like discover API does IIRC

Reply:
The reason I removed this mapping is that now, all queries end up in a bulk query with a single referrer. It might even be easier to debug now, because searching for the referrer will lead you directly to the snuba call: https://github.com/getsentry/sentry/pull/33339/files#diff-9768b861d30daef14443451ae5e87b0352a6229bef88dfc9722fd06bad13afe1R469-R479.

But I agree that the referrer itself does not say much about the intention of the query, since that depends on client input.